Lib/pickletools.py

```diff
@@ -1668,7 +1679,9 @@ def __init__(self, name, code, arg,
       Stack before:  ... pydict key value
       Stack after:   ... pydict
 
-      where pydict has been modified via pydict[key] = value.
+      where pydict has been modified via pydict[key] = value. Note that any
```

Comment:
Should not we change "dict" to "mapping" in the description? And "list" to "sequence" in the description of APPEND?